pkg/commands/resolver.go

```diff
@@ -99,6 +99,10 @@ func gobuildOptions(bo *options.BuildOptions) ([]build.Option, error) {
 	if bo.DisableOptimizations {
 		opts = append(opts, build.WithDisabledOptimizations())
 	}
+	switch bo.SBOM {
+	case "none":
+		opts = append(opts, build.WithDisabledSBOM())
```

Comment:
(not blocking) I think we'll eventually want this to be `WithSBOM(sbomImpl)`, where we'll have a `nopSBOM`, `spdx`, `cycloneDX` and a `NewMultiSBOM(sbom1, sbom2)` impl to support multiple selection.

Reply:
We have `withSBOMer` which is internal for now, we can just expose that.

Reply:
LMK if you want to expose this ~now?

Reply:
We'll do it in a future PR.